examples/cp-all-in-one-cloud/ccloud-generate-env-vars.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+###############################################################################
+# Overview:
+# This code reads the Confluent Cloud configuration in $HOME/.ccloud/config
+# and writes the ENV variables used by Docker Compose to a file called 'delta_configs/env.delta'
+#
+# Reference: https://github.com/confluentinc/quickstart-demos/blob/5.0.0-post/ccloud/ccloud-generate-cp-configs.sh
+#
+###############################################################################
+
+
+# Confluent Cloud configuration
+CCLOUD_CONFIG=$HOME/.ccloud/config
+if [[ ! -e $CCLOUD_CONFIG ]]; then
```

Comment:
`-f` is better since it checks for a regular file, `-e` is true for directories and weird file types (pipes, devices, etc)

Reply:
Ack